nnfabrik/main.py

```diff
@@ -351,3 +351,99 @@ class Seed(dj.Manual):
     definition = """
     seed:   int     # Random seed that is passed to the model- and dataset-builder
     """
+
+
+def custom_nnfabrik(
```

Comment:
Will that still be renamed or do we leave it like that?

Reply:
I'd be happy with a better name but don't really have strong feeling towards any name at the moment. Suggestions?

Reply:
Going with `my_nnfabrik`